backfiller/backfiller/main.py

```diff
@@ -26,6 +28,29 @@
 	['remote', 'channel', 'quality', 'hour'],
 )
 
+hours_backfilled = prom.Counter(
+	'hours_backfilled',
+	'Number of hours successfully backfilled',
+	['remote', 'channel', 'quality'],
+)
+
+hash_mismatches = prom.Counter(
+	'hash_mismatches',
+	'Number of segments with hash mismatches',
+	['remote', 'channel', 'quality', 'hour'],
+)
+
+get_node_errors = prom.Counter(
```

Comment:
for consistency, can we call this variable `node_fetch_errors`?

Reply:
actually, let's call it `node_list_errors`, and also change the line below. i think that's clearer as 'node fetch' can be confused with fetching segments from nodes